syntax="proto2";
package Proto.Config;

message HostId {
    required uint32 host_id = 1;
    required string host_ip = 2; //ip
    required uint32 host_port = 3; //port
    required uint32 server_port = 4; //ÒµÎñ¶Ë¿Ú
}

message PaxosGroup {
    required uint32 group_id = 1;
    repeated HostId hostids = 2; //
    optional string storage_dir = 4;
}

message ServerConfig {
    repeated PaxosGroup groups = 1;
};